src/main/java/step1/util/InputUtil.java

```diff
@@ -0,0 +1,16 @@
+package step1.util;
+
+import java.util.Scanner;
+
+public class InputUtil {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public static String returnString(String question) {
+        System.out.println(question);
+        String result = scanner.next();
+        if (result.contains("\\n")) {
+            result = result.replace("\\n", "\n");
+        }
```

Comment:
다음과 같이 변경하면 되지 않을까요 :) ?
```suggestion
        String result = scanner.nextLine();
```